src/Augurk/Extensions/RavenDbServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Augurk.Api;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Embedded;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class RavenDbServiceCollectionExtensions
+    {
+        public static void AddRavenDb(this IServiceCollection services)
+        {
+            var serverOptions = new ServerOptions
+            {
+                DataDirectory = Path.Combine(Environment.CurrentDirectory, "data")
+            };
+
+            var databaseOptions = new DatabaseOptions("FeatureStore")
```

Comment:
If we're breaking from the old version anyway.... is FeatureStore still the most logical name?
We've been adding al sorts of documents to it, most not being Features...

Reply:
Good point. I'm happy to change it to something else. Any suggestions? Perhaps just AugurkStore?